_topics/en/soft-fork-activation.md

```diff
@@ -0,0 +1,437 @@
+    needed to wait for one fork to activate before a second fork could
+    be started.
+
+  - **Inability to fail:** ISM attempts didn't have expiration dates.
+    Once node software was released that waited for the activation
+    signal, all nodes from that point needed to continue monitoring for
+    that signal until activation occurred.  There was no way to decide
+    the soft fork was no longer needed.
+
+  - **Loose measurements and unpredictable activation times:** ISM
+    activated whenever 950/1000 blocks signaled readiness.  However,
+    because Bitcoin block production is stochastic, a lucky set of
+    miners with much less than 95% of the total network hashrate could
+    sometimes produce 950 of the last 1,000 blocks, leading to premature
+    activation.  Additionally, not knowing the exact activation time in
+    advance meant it was difficult for protocol developers, merchant
+    system administrators, and mining pool operators to be available in
+    the hours shortly after activation occurred in case there were any
+    problems that needed a quick response.
+
+  BIP9 versionbits attempted to solve these problems by using the block
+  header version field as a bitfield.  Bits in the field were used for
+  signaling only---never to reject a block as invalid---and they could
+  be set in parallel.  Measurements were taken only once every 2,016
+  blocks, minimizing the window during which a small percentage of
+  hashrate would need to be lucky to project a false impression of 95%
+  signaling.  Finally, when the 95% signaling threshold was reached,
+  there would be a 2,016 block (roughly two week) "lock in" period until
+  activation occurred to allow people to prepare.  If the signaling
+  threshold wasn't reached by a timeout date, the attempt ended,
+  allowing the unused code to be removed from later node software
+  releases.
+
+  This activation method was first tested with the soft fork that added
+  [BIP68][] consensus-enforced sequence numbers, [BIP112][]
+  `OP_CHECKSEQUENCEVERIFY`, and [BIP113][] nLockTime enforcement by
+  median time past.  The fork quickly made it to the locked-in phase,
+  and then proceeded automatically to activation.
+
+  #### [2016-7] BIP9, BIP148, and BIP91: the BIP141/143 segwit activation
+
+  The segwit soft fork was implemented to use the [BIP9][] activation
+  method.  A few miners quickly began signaling readiness but support
+  peaked far below the needed 95% threshold.  Some Bitcoin users felt
+  miners were illegitimately delaying a useful new feature and supported
+  a mandatory activation mechanism specified in [BIP148][].  The final
+  form of BIP148 specified rejecting any
+  blocks that didn't signal support for BIPs 141/143 starting on a
+  certain date.
+
+  BIP148 created the risk of a consensus failure.  If miners failed to
+  signal BIP9 support for segwit, and most users continued to consider those
```

Comment:
If "signal BIP148 support for segwit" works just as well as "signal BIP9 support for segwit" I'd suggest dropping the BIP reference and just having "signal support for segwit"